frontend/components/Domain/Recipe/RecipePage/RecipePage.vue

```diff
@@ -1,5 +1,20 @@
 <template>
   <div>
+    <v-snackbar
+      v-model="snackbar"
+      :timeout="0"
+      top
+    >
+      {{ snackbarText }}
```

Comment:
Instead of returning the string from script you can just use this:
```suggestion
      {{ $tc("general.unsaved-changes") }}
```